CHANGELOG.md

```diff
@@ -61,6 +61,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
     ([#655](https://github.com/cyverse/atmosphere/pull/655))
   - Reincluded/fixed broken tests
     ([#657](https://github.com/cyverse/atmosphere/pull/657))
+  - In application_to_provider,py, determine correct upload method based
+    on Glance client version specified in main function
+    ([#667](https://github.com/cyverse/atmosphere/pull/667))
```

Comment:
Doesn't require change, but Its not relevant that the `version is specified in main function` at the change log level. My hope for the CHANGELOG is that its terse and gives you just enough to look into the pr you need.